assume/units/powerplant.py

```diff
@@ -147,9 +147,9 @@ def calculate_energy_operational_window(
         min_power, max_power = self.calculate_min_max_power(start, end)
 
         # adjust for ramp down speed
-        min_power = max(current_power - self.ramp_down, min_power)
+        min_power = max(-self.ramp_down, min_power)
```

Comment:
why is this like that? Why is current power not taken into account now?

Reply:
because min_power/max_power from calculate_min_max_power is also relative.
This is needed so that we are not bidding twice our total power with two markets